### DISCUSSION TITLE
SHA 2.2

### DISCUSSION THREAD

**Discussion by jpsouplet-ADEO** [edenhill/librdkafka#4314] [Q&A]
Could You please create the tag for 2.2 version to view the sha relative to the version ? thanks

**Comment by mensfeld** [comment]
Waiting on it as well.

**Comment by abrisco** [comment]
@emasab I would like this as well. Is there any docs or guidance on release cadence?

**Comment by davidblewett** [comment]
Ditto. We're interested in the header metadata fix and the sticky assignor fixes.

**Comment by emasab** [comment]
We're targeting a quarterly release cadence, with optional patch releases in between. We'll create v2.2.0-RC1 in the next few days and a second RC later. The release will be at the end of June.

**Reply by abrisco** [reply]
Awesome, Thank you for the info and quick response!
